lib/classes/PluginManager.js

```diff
@@ -99,9 +99,15 @@ class PluginManager {
 
   loadPlugins(plugins) {
     plugins.forEach((plugin) => {
-      const Plugin = require(plugin); // eslint-disable-line global-require
+      try {
+        const Plugin = require(plugin); // eslint-disable-line global-require
 
-      this.addPlugin(Plugin);
+        this.addPlugin(Plugin);
+      } catch (error) {
```

Comment:
Wouldn't it be better to rethrow here? If a plugin is not found, it is imo important that Serverless stops. Otherwise it would not be possible to detect misconfigurations on automated build servers in case a developer checks in a wrong (not working) configuration.
Then the build artifacts would be broken and in the worst case the issue is detected on deployment or even on use of the service. That should not happen.